windows/src/engine/keyman/KeymanTrayIcon.pas

```diff
@@ -650,10 +650,13 @@ procedure TCustomKeymanTrayIcon.SetBalloonIcon(const Value: THandle);
 
 procedure TCustomKeymanTrayIcon.SetDefaultIcon;
```

Comment:
It would be better to have a procedure called SetIcon that will set either the Application Icon or the Keyboard supplied Icon based on a bool value. That bool value is ultimately set by a configuration value in the `Keyman Configuration->Options` .